### PR TITLE
*: update the documentation to the latest version

### PR DESCRIPTION
**Pull request by rleungx**
### What problem does this PR solve? 
The documentation is outdated.

### What is changed and how it works?
This PR updates the documentation and fixes some comments

### Check List 

Tests 

 - No code

### CODE REVIEW

File: tools/pd-ctl/README.md [hunk 1]
```diff
@@ -98,50 +98,60 @@ Usage:
...
+    "enable-location-replacement": "true",
+    "enable-make-up-replica": "true",
+    "enable-one-way-merge": "false",
+    "enable-remove-down-replica": "true",
+    "enable-remove-extra-replica": "true",
+    "enable-replace-offline-replica": "true",
     "high-space-ratio": 0.6,
     "hot-region-cache-hits-threshold": 3,
-    "hot-region-schedule-limit": 2,
+    "hot-region-schedule-limit": 4,
```
Comment (lhy1024) [inline tools/pd-ctl/README.md]: where do we update?

File: tools/pd-ctl/README.md [hunk 2]
```diff
@@ -98,50 +98,60 @@ Usage:
   "replication": {
     "location-labels": "",
     "max-replicas": 3,
-    "strictly-match-label": "true"
+    "strictly-match-label": "false"
```
Comment (lhy1024) [inline tools/pd-ctl/README.md]: which pr modify it？

File: tools/pd-ctl/README.md (hunk 1 above)
Comment (rleungx) [reply tools/pd-ctl/README.md]: Both of them are the default value, but we didn't update the doc when PD was merged.

**Pull request merged**